### PR TITLE
move toLowerCase into Filter constructor

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
subj

### CODE REVIEW

File: src/core/renderers/webgl/managers/FilterManager.js [hunk 1]
```diff
@@ -327,7 +327,7 @@ export default class FilterManager extends WebGLManager
         // TODO Cacheing layer..
         for (const i in uniformData)
         {
-            const type = uniformData[i].type.toLowerCase();
+            const type = uniformData[i].type;
```
Comment (bigtimebuddy) [inline src/core/renderers/webgl/managers/FilterManager.js]: `const {type} = uniformData[i];`?

Comment (ivanpopelyshev) [reply src/core/renderers/webgl/managers/FilterManager.js]: Seems like pointless change to me.

Comment (bigtimebuddy) [reply src/core/renderers/webgl/managers/FilterManager.js]: Only a suggestion. less code

Comment (ivanpopelyshev) [reply src/core/renderers/webgl/managers/FilterManager.js]: ok, next time when i see multiple of these :)

**Pull request merged**